### PR TITLE
13210 Write tests for Draft and Final EIS package

### PR DESCRIPTION
**Pull request by CeciliaCY**
### Summary
13210 Write tests for Draft and Final EIS package

#### Task/Bug Number
Close [AB#13210](https://dcp-paperless.visualstudio.com/ZAP%20Portals/_workitems/edit/13210)

### Technical Explanation
Same as #822

### CODE REVIEW

File: client/mirage/factories/package.js [hunk 1]
```diff
@@ -80,6 +80,14 @@ export default Factory.extend({
     dcpPackagetype: 717170002,
   }),
 
+  deis: trait({
+    dcpPackagetype: 717170013,
+  }),
+
+  feis: trait({
+    dcpPackagetype: 717170003,
+  }),
```
Comment (allthesignals) [inline client/mirage/factories/package.js:89]: awesome

File: client/tests/acceptance/user-can-edit-deis-test.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    this.server.create('project', {
+      packages: [
+        this.server.create('package', 'done', 'deis'),
+      ],
+    });
+
+    await visit('/projects');
+    assert.equal(findAll('[data-test-projects-list="to-do"] [data-test-my-project-list-item]').length, 1);
+
+    assert.equal(findAll('[data-test-projects-list="done"] [data-test-my-project-list-item]').length, 1);
```
Comment (allthesignals) [inline client/tests/acceptance/user-can-edit-deis-test.js:39]: Just a comment, but it's sometimes a stronger assertion to look for some specific text, perhaps a fake project name.

File: client/tests/acceptance/user-can-edit-deis-test.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+    assert.dom('[data-test-project-dcpName]').hasText('(P2018M0268)');
+
+    assert.dom('[data-test-project-dcpBorough]').containsText('Bronx');
+
+    assert.dom('[data-test-package-statuscode]').containsText('Package Preparation');
+
+    assert.dom('[data-test-package-notes]').containsText('Some instructions from Planners');
+
+    assert.dom('[data-test-attached-documents]').exists();
```
Comment (allthesignals) [inline client/tests/acceptance/user-can-edit-deis-test.js:82]: Love these assertions!

**Review by allthesignals — APPROVED** [review]
TY for adding tests!

**Comment by CeciliaCY** [comment]
Thanks @allthesignals

**Pull request merged**